src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -375,6 +375,20 @@ configurationRegistry.registerConfiguration({
 			type: 'boolean',
 			description: nls.localize('task.quickOpen.skip', "Controls whether the task quick pick is skipped when there is only one task to pick from."),
 			default: false
-		}
+		},
+		'task.saveBeforeRun': {
+			markdownDescription: nls.localize(
+				'task.saveBeforeRun',
+				'Configures whether to save all editors before running a task, running it without saving or prompting the option every time is run'
```

Comment:
'Configures whether all editor will be saved before running a task.` should be enought.

Reply:
And for the enum descriptions:
'Always saves all editors before running.'
'Never saves editors before running.'
'Prompts whether to save editors before running.'